cfme/utils/template/virtualcenter.py

```diff
@@ -18,8 +18,10 @@ class VMWareTemplateUpload(ProviderTemplateUpload):
     def upload_template(self):
         cmd_args = [
             "ovftool --noSSLVerify",
-            # prefer the datastore from template_upload
-            "--datastore={}".format(self.provider_data.provisioning.datastore),  # move later
+            # Deploy on the datastore for provision, move to template datastore later
+            # JUWATTS CHANGE THIS
```

Comment:
Change this when? Why? Linked to other PR's? GH issues?

Reply:
This was due to #8609 , rebased so this is resolved.